test/test_manager.py

```diff
@@ -1198,3 +1198,18 @@ def assert_dd_appeared():
         assert manager_nospawn.c.eval("self.core.wmname") == (True, "LG3D")
     assert "dd" in manager_nospawn.c.groups()["S"]["windows"]  # First dropdown persists
     assert "dd" in manager_nospawn.c.groups()["1"]["windows"]  # Second orphans to group
+
+
+class CommandsConfig(Config):
+    screens = [
+        libqtile.config.Screen(
+            bottom=libqtile.bar.Bar([libqtile.widget.Systray()], 20),
+        )
+    ]
+
+
+@pytest.mark.parametrize("manager", [CommandsConfig], indirect=True)
+def test_windows_from_commands(manager):
+    manager.test_window("one")
+    assert len(manager.c.items("window")) == 2
+    assert len(manager.c.windows()) == 1
```

Comment:
Is this a bit confusing? The difference between getting 2 and 1 here is because the `Bar` is excluded from the second.

It might be good to assert that `Systray` is not in either of `items("window")` and `windows()`.